torch_xla/__init__.py

```diff
@@ -129,4 +129,5 @@ def _init_xla_lazy_backend():
 
 atexit.register(_prepare_to_exit)
 _apply_patches()
-_init_xla_lazy_backend()
+if os.environ.get('INIT_LAZY_BACKEND', '1') != '0':
```

Comment:
@JackCaoG do you think we could add some comments here explaining why and what this change / flag does for future reference?

Reply:
this is a 1.13 only quick hack, I will fix it properly in the master.

Reply:
ok added a quick comment.

Reply:
Gotcha, thanks @JackCaoG